library/Icingadb/Util/PerfDataFormat.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+/* Icinga DB Web | (c) 2021 Icinga GmbH | GPLv2+ */
```

Comment:
Please change this to `GPLv2` (Remove the plus). I know, the other PerfData files also use it, but it's wrong as well.